dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/CompanyInformation.kt

```diff
@@ -0,0 +1,22 @@
+package org.dataland.datalandbackend.model
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import java.math.BigDecimal
+import java.time.LocalDate
+
+/**
+ * --- API model ---
+ * Class for defining the request body of a post company request
+ * @param companyName name of the company
+ * @param headquarters city where the headquarters of the company is located
+ * @param industrialSector sector in which the company operates
+ * @param marketCap For publicly traded companies: The total monetary value of the outstanding shares of the company
+ * @param reportingDateOfMarketCap date to which the market cap value refers
+ */
+data class CompanyInformation(
+    @field:JsonProperty(required = true) val companyName: String = "",
+    @field:JsonProperty(required = true) val headquarters: String,
+    @field:JsonProperty(required = true) val industrialSector: String,
```

Comment:
finde ich vom wording her nicht ganz so schön da Industrie eine gewisse Sichtweise impliziert (Fabriken und so), warum nicht einfach "sector". Es könnten ja sonst Einträge geben wie industrialSector = Farming oder industrialSector = Real Estate oder ähnliches. Das passt irgendwie nicht für meinen Geschmack.

Reply:
Antwort Emanuel: 

"Industry" ist im Englischen nicht ganz vergleichbar mit dem Deutschen (Finanzbranche wird z.B. finance industry genannt etc.).
Das Wort "industrial sector" kam nach einer längeren Dictionary-Recherche zustande.
Aber sector alleine stand eigentlich auch immer in so gut wie jedem Dictionary mit dabei.
Also finde ich auch gut. Und kürzer.

Auch hier der Hinweis (und in unter einem JIRA-Ticket hatten wir das ja auch mal festgehalten), dass es in dem figma-Entwurf mal so mal so war (einmal industry, einmal sector).

Reply:
ja das ist mir bewusst aber ich würde hier sector bevorzugen weil es einfach und klar ist

Reply:
Ok, passt für mich

Reply:
__Beginne zu fixen

Reply:
fixed